extensions/2.0/Khronos/KHR_materials_diffuse_transmission/README.md

```diff
@@ -0,0 +1,260 @@
+    ior = 1.5,
+    base = mix(
+      diffuse_brdf(baseColor),
+      specular_btdf(α = roughness^2) * baseColor,
+      transmission),
+    layer = specular_brdf(α = roughness^2)
+  )
+```
+
+Since the diffuse BTDF does not have controls for roughness, the roughness parameter acts only on the reflective part of the surface. By decoupling the reflection and transmission parts it is possible to configure materials which have a smooth reflection and a diffuse transmission, as shown in image below (right).
+
+<figure style="text-align:center">
+<p float="left">
+ <img src="./figures/transmission-translucency_thin.jpg" width="40%" />
+ <img src="./figures/transmission-translucency_thin_angle.jpg" width="39.5%" />
+</p>
+<sub><figcaption><em>Emissive sphere behind material sample. Left: Opaque diffuse. Middle: Rough transmission. Right: Diffuse transmission.</em></figcaption></sub>
+</figure><br>
+
+If `KHR_materials_transmission` is used in combination with `KHR_materials_diffuse_transmission`, the transmission effect overrides the diffuse transmission effect.
+
+We can formalize this behavior by combining the two cases from above
+```
+dielectric_brdf =
+  fresnel_mix(
+    ior = 1.5,
+    base = mix(
+      diffuse_bsdf,
+      specular_btdf(α = roughness^2) * baseColor,
+      transmission),
+    layer = specular_brdf(α = roughness^2)
+  )
+
+diffuse_bsdf = mix(
+    diffuse_brdf(color = baseColor),
+    diffuse_btdf(color = diffuseTransmissionColor),
+    diffuseTransmission)
+```
+
+<figure style="text-align:center">
+ <img src="./figures/star2.jpg" width="500" />
+<sub><figcaption><em>95% transmissive, 5% translucent</em></figcaption></sub>
+</figure>
+
+
+### KHR_materials_volume
+If `KHR_materials_diffuse_transmission` is combined with `KHR_materials_volume`, a diffuse transmission BTDF describes the transmission of light through the volume boundary. The object becomes translucent. The roughness parameter only affects the reflection. Scattering and absorption inside the volume are computed as defined in `KHR_materials_volume` and `KHR_materials_sss`, so the random walk through the volume is not affected by the surface BSDF.
+
+<figure style="text-align:center">
+<img src="./figures/translucent-roughness.png"/>
+<sub><figcaption><em>Translucent sphere with varying roughness. From left to right: 0.0, 0.2, 0.4.</em></figcaption></sub>
+</figure>
+
+For comparison, below is the result for the same scene with `KHR_materials_transmission` instead of `KHR_materials_diffuse_transmission`. Here, a refractive microfacet BTDF describes the transmission of light through the volume boundary. The refraction occurs on microfacets, and thus the roughness parameter affects both reflection and transmission.
+
+<figure style="text-align:center">
+<img src="./figures/transmissive-roughness.png"/>
+<sub><figcaption><em>Transmissive sphere with varying roughness. From left to right: 0.0, 0.2, 0.4.</em></figcaption></sub>
+</figure>
+
+### KHR_materials_sss
```

Comment:
Should this discussion move to the SSS extension?

Reply:
Just an update to my comment above. On one of our TSG calls, we did confirm that this section belongs in the KHR_materials_sss extension.

Reply:
Removed all references to KHR_materials_sss with 70ff0da. Will add the relevant infos to KHR_materials_sss.